common/Log.hpp

```diff
@@ -260,13 +260,13 @@ static constexpr std::size_t skipPathPrefix(const char (&s)[N], std::size_t n =
 
 #ifdef __ANDROID__
 
-#define LOG_LOG(LOG, PRIO, LVL, STR)                                                               \
+#define LOG_LOG(LOG, NAME, PRIO, LVL, STR)                               \
     ((void)__android_log_print(ANDROID_LOG_DEBUG, "coolwsd", "%s %s", LVL, STR.c_str()))
 
 #else
 
-#define LOG_LOG(LOG, PRIO, LVL, STR)                                                               \
-    LOG.log(Poco::Message(LOG.name(), STR, Poco::Message::PRIO_##PRIO))
+#define LOG_LOG(LOG, NAME, PRIO, LVL, STR)                           \
+    LOG.log(Poco::Message(NAME, STR, Poco::Message::PRIO_##PRIO))
```

Comment:
How does this remove a Poco dependency  ? =) I was hoping we could come up with a new enumeration to replace Poco::Message::PRIO_ and use LVL as the enumeration value - switch that from a string in the macro and stringify that for where it is used to simplify. Not sure what the Name is good for (?)

Reply:
Oh, LOG_ANY does not have a name (empty string), I cannot call macro LOG_LOG without a name , but the LVL enumeration it is a good idea, reworking ...